server/src/test/java/org/elasticsearch/action/fieldcaps/FieldCapabilitiesRequestTests.java

```diff
@@ -67,6 +87,11 @@ protected FieldCapabilitiesRequest mutateInstance(FieldCapabilitiesRequest insta
         });
         mutators.add(request -> request.setMergeResults(request.isMergeResults() == false));
         mutators.add(request -> request.includeUnmapped(request.includeUnmapped() == false));
+        mutators.add(request -> request.nowInMillis(request.nowInMillis() != null ? request.nowInMillis() + 1 : 1L));
+        mutators.add(
+            request -> request.indexFilter(request.indexFilter() != null ? request.indexFilter().boost(2) : QueryBuilders.matchAllQuery())
+        );
+        mutators.add(request -> request.runtimeFields(Collections.singletonMap("other_key", "other_value")));
 
```

Comment:
would it make sense to add a test for the toXContent output of the request if we don't test it yet?

Reply:
sure, will do